Tasks/VsTest/task.loc.json

```diff
@@ -165,7 +165,14 @@
     "PowerShell": {
       "target": "$(currentDirectory)\\VSTest.ps1",
       "argumentFormat": "",
-      "workingDirectory": "$(currentDirectory)"
+      "workingDirectory": "$(currentDirectory)",
+      "platforms": [
+        "windows"
+      ]
+    },
+    "Node": {
```

Comment:
Actually it depends on the agent. In the coreclr agent if platform is not given like above node implementation will be choosen